src/xcode/ENA/ENA/Source/Scenes/RiskLegend/Cell/RiskLegendDotBodyCell.swift

```diff
@@ -5,7 +5,47 @@
 import UIKit
 
 class RiskLegendDotBodyCell: UITableViewCell {
-
-	@IBOutlet weak var dotView: UIView!
 	
+	// MARK: - Init
+	
+	required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+	
+	// MARK: - Overrides
+	
+	override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+		super.init(style: style, reuseIdentifier: reuseIdentifier)
+		// dotView
+		dotView = UIView()
+		dotView.backgroundColor = .enaColor(for: .riskHigh)
+		dotView.layer.cornerRadius = 8
+		dotView.layer.masksToBounds = true
+		dotView.translatesAutoresizingMaskIntoConstraints = false
+		contentView.addSubview(dotView)
+		NSLayoutConstraint.activate([
```

Comment:
Would be nice to group the two 
`NSLayoutConstraint.activate(` into one. For maximum performance 🐎